src/main/java/ladder/domain/UserNames.java

```diff
@@ -0,0 +1,52 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class UserNames {
+    static final int MIN_SIZE = 2;
```

Comment:
```suggestion
    private static final int MIN_SIZE = 2;
```
외부에서 사용하지 않는 상수는 private으로 막아주세요

Reply:
아래와 같이 테스트 코드 내에서 예외 메시지를 확인할 때 해당 상수를 사용하고 있어서 private을 사용하지 않았습니다.
테스트를 작성할 때 상수 값도 물론 중요하겠지만 기능이 잘 작동하는지 확인하는 게 중요하다고 생각해서 상수를 테스트 코드에서 import하여 사용해 주었습니다.

실제로 이전 미션에서 상수 값을 format해서 사용하지 않다가 프로덕션 코드와 테스트 코드의 숫자 값이 달라져서 예외가 실패했던 경험이 있어 이렇게 작성하였는데, 이 부분에 대해서 오리는 어떻게 생각하시는지 궁금합니다.

테스트 코드는 상수 값을 사용하지 않고 바로 `참여자는 2명 이상이어야 합니다.`와 같이 기재해 주는 것이 더 좋을까요? 🤔

```java
@DisplayName("사용자가 2명 미만이면 예외가 발생한다.")
@Test
void createUserNamesByLowerSize() {
    assertThatIllegalArgumentException()
            .isThrownBy(() -> UserNames.of(List.of("a")))
            .withMessage(String.format(
                    "참여자는 %d명 이상이어야 합니다.", MIN_SIZE
            ));
}
```

Reply:
네 상수를 가져다쓰기보다는 아래에서 이야기한대로 테스트의 기대 결과값은 로직이 있거나 다른 곳에 의존하지 않았으면해요. 경험하신 내용이 구체적으로 어떤 상황인지는 알아야 알겠지만, 프로덕션 코드의 리팩터링이 아닌 기능 변경은 테스트코드의 변화에 영향을 끼치는건 자연스러운 것 같아요.

(물론 사람마다 테스트하는 방식은 다를순있습니다.)